src/mesh/smoothing.rs

```diff
@@ -3,59 +3,48 @@ use std::collections::hash_map::{Entry, HashMap};
 use std::hash::{Hash, Hasher};
 
 use nalgebra as na;
-use nalgebra::geometry::Point3;
+use nalgebra::Point3;
 use smallvec::SmallVec;
 
 use crate::convert::{cast_u32, cast_usize};
-use crate::geometry::{Face, Geometry, NormalStrategy};
+use crate::mesh::{Face, Mesh, NormalStrategy};
 
-/// Relaxes angles between mesh edges, resulting in a smoother geometry
-///
-/// or
-///
-/// Relaxes angles between mesh edges, while optionally keeping some vertices
-/// anchored, resulting in an evenly distributed geometry optionally stretched
-/// between the anchor points
+/// Relaxes angles between mesh edges, resulting in a smoother
+/// mesh, optionally keeping some vertices anchored, resulting in
+/// an evenly distributed geometry optionally stretched between the
+/// anchor points
 ///
 /// The number of vertices, faces and the overall topology remains unchanged.
 /// The more iterations, the smoother result. Too many iterations may cause slow
-/// calculation time. In case the stop_when_stable flag is set on, the smoothing
-/// stops when the geometry stops transforming between iterations or when it
+/// calculation time. In case the `stop_when_stable` flag is set on, the smoothing
+/// stops when the mesh stops transforming between iterations, or when it
 /// reaches the maximum number of iterations.
 ///
 /// The algorithm is based on replacing each vertex position with an average
 /// position of its immediate neighbors.
 ///
-/// - `geometry` - mesh geometry to relax
-/// - `iterations` - (maximum) number of times the smoothing algorithm should
-///   relax the geometry
-/// - `fixed_vertex_indices` - indices of vertices to keep fixed during the
-///   relaxation
-/// - `stop_when_stable` - the smoothing stops when there is no change between
-///   iterations
-///
-/// returns (smooth_geometry: Geometry, executed_iterations: u32, stable: bool)
+/// Returns `(smooth_mesh: Mesh, executed_iterations: u32, stable: bool)`.
 pub fn laplacian_smoothing(
-    geometry: &Geometry,
+    mesh: &Mesh,
     vertex_to_vertex_topology: &HashMap<u32, SmallVec<[u32; 8]>>,
-    iterations: u32,
+    max_iterations: u32,
```

Comment:
_Just an explanation:_ I named this `iterations` rather than `max_iterations` because in case the `stop_when_stable` flag is off, this will be the actual number of iterations.